source/vocab/relations.ttl

```diff
@@ -444,6 +444,13 @@ prefix relsubtype: <http://id.loc.gov/vocabulary/preservation/relationshipSubTyp
     owl:inverseOf :hasSeries;
     owl:equivalentProperty bf2:seriesOf .
 
+:isIssueOf a owl:ObjectProperty;
+    # :category :pending ?
+    rdfs:label "Is issue of"@en, "nummer av"@sv;
+    rdfs:subPropertyOf :isPartOf;
+    # owl:inverseOf :hasIssue; ?
```

Comment:
We could have use for the reverse for providing a label, but that reverse should be pending or "abstract" unless we have a need to allow both directions (which does complicate consuming this data, as it would be a "denormalized" situation).

Reply:
For the "a reverse label would be nice" case, [it has been suggested](https://lists.w3.org/Archives/Public/public-rdf-wg/2012Nov/0333.html) that using a blank reverse with just the label is good enough. I like that pattern. When an actual need to use the inverse as a term in data arise, we can just mint a URI for it. Until then, it is handily "out of the way" as a bnode with a label.

Reply:
Fixed with https://github.com/libris/definitions/pull/368/commits/1416eb396e9a0a69136ec5fa57ebf2daa1c55c3e